src/Microsoft.IdentityModel.ManagedKeyVaultSecurityKey/Microsoft.IdentityModel.ManagedKeyVaultSecurityKey.csproj

```diff
@@ -9,8 +9,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <LangVersion>latest</LangVersion>
     <PackageId>Microsoft.IdentityModel.ManagedKeyVaultSecurityKey</PackageId>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net452;netstandard1.4</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.4</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net452;netstandard2.0</TargetFrameworks>
```

Comment:
I don't think that current versions of ManagedKeyVaultSecurityKey dependencies support netstandard2.0.
We should probably upgrade, as the new versions support netstandard2.0.

Reply:
I've updated the `MicrosoftAzureKeyVaultVersion `to 3.0.5 in dependencies.props and `SystemNetHttpVersion` to 4.3.4 in both dependencies.props and depedenciesTest.props.

It also seems like [`Microsoft.Azure.KeyVault`](https://www.nuget.org/packages/Microsoft.Azure.KeyVault/) has been replaced by a set of different libraries, so it may be worth switching to those as well...

Reply:
I suggest updating all references to the latest available. if not at least to the min recommended version (considering security, performance and feature updates etc.)

Reply:
@henrik-me I'm merging this PR in now, but I created a separate issue to track this: #1326